ocaml/bytecomp/bytelink.ml

```diff
@@ -203,13 +200,12 @@ let check_consistency file_name cu =
     Array.iter
       (fun import ->
         let name = Import_info.name import in
-        let crco = Import_info.crc import in
-        let cuo = Import_info.Intf.impl import in
+        let nonalias = Import_info.Intf.nonalias import in
         interfaces := name :: !interfaces;
-        match crco with
+        match nonalias with
           None -> ()
-        | Some crc ->
-            Consistbl.check crc_interfaces name cuo crc file_name)
+        | Some (sort, crc) ->
```

Comment:
Sometimes this component seems to be a "sort" and sometimes a "kind". Probably worth picking one name consistently.